python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -737,7 +738,7 @@ def run(self, subExposure, expandedSubExposure, fullBBox,
         subExp1 = templateExposure.Factory(templateExposure, expandedSubExposure.getBBox())
 
         # Prevent too much log INFO verbosity from DecorrelateALKernelTask.run
-        logLevel = self.log.getLevel()
+        logLevel = self.log.level
         self.log.setLevel(self.log.WARNING)
```

Comment:
I really need to add the logging context manager to the `utils` package so that the log level resets when the task below completes. I'm not overly happy with how we handle this since it really is saying that this task is too verbose by default and it's impossible to ever see the info messages. As written, if someone sets the log level to DEBUG in pipetask they won't get any debug messages out of this task and that seems wrong.

Reply:
I completely agree that code that forces the log level like this is problematic. This is code that is not used outside of the unit tests, for now at least, and I just wanted to change this line to fix a warning while building the package. This particular task will need to be refactored if we want to use it in production.